pkgs/development/libraries/fmt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, cmake }:
+
+let
+  version = "5.2.1";
+in
+
+stdenv.mkDerivation rec {
+  name = "fmt-${version}";
+  src = fetchurl {
```

Comment:
We prefer to use fetchFromGithub (it only downloads a tar file)

Reply:
Changed.